compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/fragments/EvaluatorFragmentInfo.kt

```diff
@@ -22,5 +22,10 @@ import org.jetbrains.kotlin.descriptors.FunctionDescriptor
 class EvaluatorFragmentInfo(
     val classDescriptor: ClassDescriptor,
     val methodDescriptor: FunctionDescriptor,
-    val parameters: List<DeclarationDescriptor>
-)
\ No newline at end of file
+    val parameters: List<EvaluatorFragmentParameterInfo>,
```

Comment:
Could you please keep the constructor with the old signature, so that the current plugin code will compile (and if possible, work somehow)? Otherwise I can't merge the changes because IDE plugin build is a part of the verification check for all builds.

(Alternatively, I could wait until https://github.com/JetBrains/intellij-community/pull/1839 is approved and somehow agree to merge them ~"atomically", but that seems a bit more complicated.)

Reply:
Oh! I had not realised this complication! Yes, I will keep in the other constructor, and keep this in mind in the future!

Reply:
Hmm, so this is not immeadiately obvious to me. The new and old constructor signatures can't coexist: they cause a platform declaration clash due to the erasure of generics on List.